doc/project/project_roles.rst

```diff
@@ -314,15 +314,37 @@ Release Activity
 Merge Criteria
 ++++++++++++++
 
-* All continuous integration checks have passed
+* Minimal of 2 approvals:
+
+  * A collaborator from the same subsystem.
+  * Alternately another maintainer of another subsystem
+  * Approval by the assignee
+
+* Four eye principle on the organisation level.
+
+  * Changes or additions to common and shared code shall have reviews from
+    different organisations (at least one review and approval from a different
+    submitters' organisation).
+  * Changes or additions to hardware support (driver, SoC, boards) shall at
+    least have the merger be from a different organisation.
+  * Release engineers shall not merge code changes originating from the same
+    organisation if all other reviewers are also from the same organisation.
```

Comment:
Is that:
-If all approvals of a change come from the same organisation, it can only be merged by a release engineer coming from another organisation.
-A change coming from one organisation needs at least an approval from another organisation to be merged

?